nautilus_trader/adapters/interactive_brokers/historic/client.py

```diff
@@ -59,11 +60,12 @@ def __init__(
         port: int = 7497,
         client_id: int = 1,
         market_data_type: MarketDataTypeEnum = MarketDataTypeEnum.REALTIME,
+        log_level: str = "INFO",
```

Comment:
We could also use `LogLevel` here, which could then be passed straight through to the `Logger`?

Reply:
I initially had it that way but then thought it might be more user-friendly to make it a string -- similar to `log_level` in `LoggingConfig`.

Reply:
Yep, I had a similar thought too. The only reason its that way for `LoggingConfig` is so that it can be serialized, ideally we'd use a stronger type (and could add a msgspec encoding/decoding hook as such). I thought it might save you the trouble of importing the `log_level_from_str` function.

In any case, this works too - the above just another approach :ok_hand:

Reply:
> The only reason its that way for `LoggingConfig` is so that it can be serialized

@cjdsellers enums are supported now in `msgspec` by default so you may plan for strict typing in NautilusConfig.